### ISSUE TITLE
An unusable link 	before "Cog angle" button

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1948]
URL: http://clarity-ks.surge.sh/alerts

Reproduced Steps:
1. Go to Alerts page
2. Use tab button browse the page.

Actual Results:
When use tab button browse web page (before "Cog angle" button), there is an unusable link can be focused on.

Expected Results:
There is no unusable link in the web page.

Violation of Section 508 Standards: 
1194.21(c) A well-defined on-screen indication of the current focus shall be provided that moves among interactive interface elements as the input focus changes. The focus shall be programmatically exposed so that assistive technology can track focus and focus changes. 

User Impact: Blind.

Note:
This issue only occurred with IE.

NOTE: not sure what is going on here.

**Comment by mathisscott** [comment]
This is just decorative for the demo app.
